### PR TITLE
PR responses

### PR DESCRIPTION
**Pull request by wpride**
responses to https://github.com/dimagi/formplayer/pull/16/files#r51883783

### CODE REVIEW

File: src/main/java/hq/RestoreUtils.java [hunk 1]
```diff
@@ -71,8 +71,7 @@ public static FormDef loadInstance(InputStream formInput,
         // weak check for matching forms
         if (!savedRoot.getName().equals(templateRoot.getName()) ||
                 savedRoot.getMult() != 0) {
-            System.out.println("Instance and template form definition don't match");
-            return null;
+            throw new Exception("Instance and template form definition don't match");
```
Comment (snopoke) [inline src/main/java/hq/RestoreUtils.java]: Fine for now but a custom exception would be better.

File: src/main/java/util/Constants.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    public final static String URL_SUBMIT_FORM = "submit";
+    public final static String URL_GET_INSTANCE = "get_instance";
+    public final static String URL_EVALUATE_XPATH = "evaluate_xpath";
+    public final static String URL_NEW_REPEAT = "new_repeat";
+    public final static String URL_DELETE_REPEAT = "delete_repeat";
+    public final static String URL_FILTER_CASES = "filter_cases";
+    public final static String URL_SYNC_DB = "sync_db";
+    public final static String URL_LIST_SESSIONS = "sessions";
+    public final static String URL_GET_SESSION = "get_session";
+}
```
Comment (benrudolph) [inline src/main/java/util/Constants.java]: :tight:

**Comment by benrudolph** [comment]
lgtm, holding off on merge because of label

**Comment by wpride** [comment]
@benrudolph merged, thanks

**Pull request merged**